app/workers/feeds/import_articles_worker.rb

```diff
@@ -4,16 +4,10 @@ class ImportArticlesWorker
 
     sidekiq_options queue: :medium_priority, retry: 10
 
-    def perform(user_ids = [])
-      user_ids = Array.wrap(user_ids)
+    def perform(user_ids = [], earlier_than = nil)
```

Comment:
In an effort to prevent us from ending up with a never-ending job accidentally, I think we should require an `earlier_than` time here.

Reply:
Makes sense, though I'm a bit conflicted about changing it because it'd result in "hacks" to select all users, something like `earlier_than: Time.at(0)` or something. Or maybe we make use `perform(nil)`  as "select everything". What do you think?

Reply:
We should never be enqueueing one of these with all users based on how long the worker takes so I think it would be fine leaving it in a state where to get all users you would have to do something unconventional or hacky. I would even be game for a comment that warns that using the worker with a lot of users can cause long runtimes even.

Reply:
Okay! I'll rework the parameters to make the time span mandatory